### DISCUSSION TITLE
myblog/asahi-linux/

### DISCUSSION THREAD

**Discussion by giscus** [jasoneckert/jasoneckert.github.io#6] [Announcements]
# myblog/asahi-linux/

My personal website and blog

https://jasoneckert.github.io/myblog/asahi-linux/

**Comment by slartybardfast** [comment]
He'll yeah borther!  I was lazily googling around about dual boot questions instead of just reading the install script when I came across this.  I snagged a mini m1 cheap on ebay recently and I might give this a whirl there or on my m1 air.  Still teaching me things 20 years later ;).  Awesome documentation as alwyas and thank you again!

**Reply by jasoneckert** [reply]
That's great - glad you enjoyed it :-)
